dapp/scripts/createTestColony.js

```diff
@@ -0,0 +1,64 @@
+// Import the prerequisites
+const { providers, Wallet } = require('ethers');
+const { default: EthersAdapter } = require('@colony/colony-js-adapter-ethers');
+const { TrufflepigLoader } = require('@colony/colony-js-contract-loader-http');
+
+// Import the ColonyNetworkClient
+const { default: ColonyNetworkClient } = require('@colony/colony-js-client');
+
+// Create an instance of the Trufflepig contract loader
+const loader = new TrufflepigLoader();
+
+// Create a provider for local TestRPC (Ganache)
+const provider = new providers.JsonRpcProvider('http://localhost:8545/');
```

Comment:
As I understand it, this line will have to change to support metamask. Here's an example:
https://dev.to/yukkurisinai/using-metamask-as-web3js-provider-2do4 

I'm okay with leaving this for just testing purposes, but we need a task for metamask integration.